dist/tools/pyterm/pyterm

```diff
@@ -241,7 +241,7 @@ class SerCmd(cmd.Cmd):
                           "out" % line)
         for tok in line.split(';'):
             tok = self.get_alias(tok)
-            self.ser.write((tok.strip() + "\n").encode("utf-8"))
+            self.ser.write((tok.strip() + "\n").decode("utf-8").encode("utf-8"))
```

Comment:
that it was not running, because my env python was pointing to 3.x
I have no idea how this behaves now, didn't test it for a while. What happens if you just drop the `encode` and only use `decode`?